tests/Unit/PointwiseFunctions/GeneralRelativity/Test_WeylTypeD1.cpp

```diff
@@ -0,0 +1,60 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
```

Comment:
Don't you need to add this test to `tests/Units/PointwiseFunctions/GeneralRelativity/CMakeLists.txt`? Otherwise, the test doesn't build or run. If you do add it, does the test still pass?